libzhlgen/test/functions/mac/1.6.13/MantisAnimation.zhl

```diff
@@ -0,0 +1,12 @@
+"554889e548c7070000000048c747080000000048c7471000000000c6071648b86d616e74":
+cleanup __amd64 std::string MantisAnimation::GetDeathSound(MantisAnimation *this);
+"554889e54157415653504889fb488b431041be900600004c8b384d01f748634b5848635360488d0c49488b04c84869caf0000000488d7c0828f30f1005????2500e8????0e004c89ffe8????0e00488b43104c8b78484d01f748634b5848635360488d0c49488b04c84869caf0000000488d7c0828f30f1005????2500e8????0e004c89ffe8????0e00488b43104c8b78184d01f748634b5848635360488d0c49488b04c84869caf0000000488d7c0828f30f1005????2500e8????0e004c89ffe8????0e00488b43104c03703048634b5848635360488d0c49488b04c84869caf0000000488d7c0828f30f1005????2500e8????0e004c89f7e8????0e0048634358488b4b1048635360488d0440488b04c14869caf00000008b84088c00000083f803751480bb620a0000":
```

Comment:
This signature is too large, maximum signature size is `512` characters.
Use some noHook's to skip chunks you don't want to match if needed and build a smaller relative match signature. (or if this is after `::GetDeathSound` in the binary you might already be good to use a smaller non-unique relative signature as long as it's the first match after `GetDeathSound`)